mirdata/datasets/compmusic_otmm_makam.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
+"""OTMM Makam Recognition Dataset Loader
```

Comment:
should this have a method to allow you to work with private audio?

Reply:
Hmm.. that's a good question! In the Zenodo repo (https://zenodo.org/record/58413#.YBLCaWPOH-b) it is said that the audio cannot be shared (maybe it cannot even be shared for research purposes), so I am not sure if there is a source from where to get the audio tracks! That would be quite essential for the index to be properly written. Should we ask Xavier and the rest of the compilers?

Reply:
The audio is accessible through [pycompmusic](https://github.com/MTG/pycompmusic) given that to researchers, who are given access to Dunya upon [registration](https://dunya.compmusic.upf.edu/user/register/).

I don't know if `mirdata` provides (or should provide) an adapter to connect to such external sources via security tokens. Either way, I think it's between the current MTGers and `mirdata` maintainers to give a decision. If there is appetite, I can share some starter code to download the relevant audio.

Reply:
hey @sertansenturk, we're already working with @genisplaja to integrate pycompmusic into mirdata, so this would definitely be a possible way to load the data.

Reply:
Neat! Then, I'll stop trying to sell sawdust to the lumber mill! :stuck_out_tongue_closed_eyes: